lenskit-core/src/main/java/org/grouplens/lenskit/data/event/Like.java

```diff
@@ -20,14 +20,120 @@
  */
 package org.grouplens.lenskit.data.event;
 
+import org.apache.commons.lang3.builder.HashCodeBuilder;
+
+import java.io.Serializable;
+
 /**
  * Generic interface representing a unary interaction.  This provides a basic, generic representation
  * of likes, clicks, purchases, and other unary interactions.  Applications may want to implement
  * their own events to represent such interactions.
+ * <p>
+ * This class can be created with {@link LikeBuilder} or the {@link #create(long, long, long)} method.
```

Comment:
I can add the two-argument reference pretty easily.

Comment above discusses why factory methods instead of constructors. We should perhaps discuss this more.

Even if most code uses the create method, the builder is used as a part of the text file input system. Each builder implements `EventBuilder`, so the definitions of common fields such as `USER` can add the field's value to any kind of event builder.